pkgs/applications/version-management/cvs/default.nix

```diff
@@ -40,10 +40,6 @@ stdenv.mkDerivation rec {
     "AR=${stdenv.cc.targetPrefix}ar"
   ];
 
-  env = lib.optionalAttrs stdenv.cc.isClang {
```

Comment:
Removing this breaks the package on darwin. Instead, could you change the condition to `stdenv.cc.isClang && lib.versionAtLeast stdenv.cc.version "16"`?

Reply:
Changed and ofborg now succeeds in building the package.